samplers/build.gradle.kts

```
@@ -9,4 +9,7 @@ otelJava.moduleName.set("io.opentelemetry.contrib.sampler")
 dependencies {
   api("io.opentelemetry:opentelemetry-sdk")
   api("io.opentelemetry:opentelemetry-semconv")
+  api("io.opentelemetry:opentelemetry-exporter-otlp")
+  api("io.opentelemetry:opentelemetry-sdk-extension-autoconfigure")
```

(only needed for tests)
```suggestion
  testImplementation("io.opentelemetry:opentelemetry-sdk-extension-autoconfigure")
```